preliz/distributions/skewnormal.py

```diff
@@ -150,6 +150,21 @@ def kurtosis(self):
             * ((delta * np.sqrt(2 / np.pi)) ** 4 / (1 - 2 * (delta**2) / np.pi) ** 2)
         )
 
+    def mode(self):
```

Comment:
This gives the wrong results for large absolute values of alpha. Not sure about the proper fix, but maybe it could be  enough to ensure the mode is no smaller than mu (for positive values of alpha) and no larger than mu (for negative values of alpha)